source/partials/next_release/_release.md.erb

```diff
@@ -1,102 +1,54 @@
-* <%= link_to_commit '6a560ee68b6c5b4950c2e7d6ea011aea6a2f6e25', 'Display pause timestamp on the dashboard when a pipeline is paused.', 5719%>
-* <%= link_to_commit '2665b54a93898b8f7421e91185ca9deec6675f27', 'Show agents assigned to environments via config repository.', 5761%>
-* <%= link_to_commit '33171234ed0a8550a9848e94030223db75bc7a73', 'Added time stamp for the last triggered maintenance mode.', 5786%>
+* Improve concistency of some error messages generated by GoCD server.
+* <%= link_to_pull_request 6036, 'Improvements to the user management page.' %>
+* <%= link_to_issue 6042, 'Allow reordering of pipelines from config repositories.' %>. This adds an optional `display_order_weight` property for pipelines defined in config repositories, which allow you to order pipelines.
+* macOS installers will no longer ship as osx `.app` files. The installers now ship as a zip file, with a shell script. We believe this provides much better flexibility and allows users to run the GoCD agent and server in a location of their choice.
 
 <h4>Bug fixes</h4>
 
-* <%= link_to_commit '9a1eaf337689b327f694fbbfe15b455d2b78b259', 'Do not remove MaterialHealthStateScope for Config Repo materials.', 5651 %>
-* <%= link_to_commit 'ba495470c154cf46aadf3ac45c67544735391c61', 'Disable job settings link for pipelines defined in Config Repo on elastic profiles page.', 5604 %>
-* <%= link_to_commit '3f2fc49422a9fb93ccb6b938dfd8ea4ed75c92a2', 'Made status report button state on elastic profiles page consistent with the  status report button on plugins page ', 5776 %>
-* <%= link_to_commit '6c4da91079e3ea11e905da14f7cf709604e7d7a5', 'Fixed environments api version 2 etag generation issue.', 5807 %>
-* <%= link_to_commit '38e68d0ca9dfce367dd1d9a380e824595cd87d34', 'Render console log containing new line character ("\n").', 5037 %>
-* <%= link_to_issue 5705, 'Fix for high CPU usage on console logs/tests page.' %>
-* <%= link_to_pull_request 5897, 'Killing agent via stop command honors the process ID' %>
-* <%= link_to_issue 5788, 'Fixed back button navigation on dashboard.' %>
+* <%= link_to_issue 5956, 'Fixed issue with environments API incorrectly rendering encrypted variables as `******` instead of the correct encrypted value.' %>
+* <%= link_to_issue 5886, 'Do not allow assigning an agent to an environment defined in config repository environment.' %>
+* <%= link_to_issue 5968, 'Fix spurious `e.map is not a function` javascript error for notifications.' %>
+* <%= link_to_pull_request 5974, "Unable to expand/collapse environments with name containing special characters." %>
+* <%= link_to_issue 5962, "GoCD shows HTTP Basic Auth dialog when after user is logged out." %>
+* <%= link_to_pull_request 6011, "Fix errors with parsing git log when coloring is turned on using `.gitconfig`." %>
+* <%= link_to_pull_request 5967, "Environments api should accept both `encrypted_value` and `value`." %>
```

Comment:
Does it give an impression that the user can provide both the properties at the same time?